### PR TITLE
fix(docs): typo fixes

### PR DESCRIPTION
**Pull request by Hello9999901**
Found and fixed some typos on the user-setup page.

### CODE REVIEW

File: docs/docs/user-setup.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ storage device. Once the flash is complete, the controller should automatically
 
 ZMK will automatically advertise itself as connectable if it is not currently connected to a device. You should be able to see your keyboard from the bluetooth scanning view of your laptop or phone / tablet. It is reported by some users that the connections with Android / iOS devices are generally smoother than with laptops, so if you have trouble connecting, you could try to connect from your phone or tablet first to eliminate any potential hardware issues.
 
-ZMK support multiple BLE “profiles”, which allows you to connect to and switch among multiple devices. Please refer to the [Bluetooth behavior](behaviors/bluetooth.md) section for detailed explanations of how to use them.
+ZMK supports multiple BLE “profiles”, which allows you to connect to and switch amongst multiple devices. Please refer to the [Bluetooth behavior](behaviors/bluetooth.md) section for detailed explanations on how to use them.
```
Comment (caksoylar) [inline docs/docs/user-setup.md:202]: Looks good to me, just a comment that "amongst" seems to be equivalent to "among" but less commonly used in American English, so I'd prefer the former.

Comment (Hello9999901) [reply docs/docs/user-setup.md:202]: Thanks! So you would prefer "amongst"?

Comment (caksoylar) [reply docs/docs/user-setup.md]: Yeah sorry, typo there. I'd prefer "among" :)

Comment (Hello9999901) [reply docs/docs/user-setup.md]: All good, changed it back haha

**Review by caksoylar — APPROVED** [review]

**Review by petejohanson — APPROVED** [review]

**Pull request merged**